src/test/java/insat/company/platform/service/MailServiceIntTest.java

```diff
@@ -184,4 +186,16 @@ public void testSendEmailWithException() throws Exception {
         mailService.sendEmail("john.doe@example.com", "testSubject", "testContent", false, false);
     }
 
+    @Test
+    public void sendInvitationMail() throws MessagingException, IOException {
```

Comment:
change the name here to the expected result, for example, shouldSendAnInviationMailSuccessfully